frontend/src/lib-components/molecules/modals/InfoModal.tsx

```diff
@@ -54,14 +54,22 @@ export default React.memo(function InfoModal({ children, ...props }: Props) {
           <Button
             appearance="inline"
             data-qa="modal-okBtn"
-            onClick={props.resolve.action}
+            onClick={(e) => {
+              e.preventDefault()
+              e.stopPropagation()
```

Comment:
Why add `preventDefault` and `stopPropagation` calls here?

Reply:
When I was clicking the modal buttons and the application row was behind them, that also got clicked and opened. This helped, but maybe only one of them is necessary.